pkg/deviceclaimingserver/enddevices/enddevices.go

```diff
@@ -42,7 +41,7 @@ type EndDeviceClaimer interface {
 	// SupportsJoinEUI returns whether the Join Server supports this JoinEUI.
 	SupportsJoinEUI(joinEUI types.EUI64) bool
 	// Claim claims an End Device.
-	Claim(ctx context.Context, req *ttnpb.ClaimEndDeviceRequest) (*ttnpb.EndDeviceIdentifiers, error)
+	Claim(ctx context.Context, joinEUI *types.EUI64, devEUI *types.EUI64, claimAuthenticationCode string, networkServerAddress string) error
```

Comment:
nit: `... joinEUI, devEUI *types.EUI64, claimAuthenticationCode, networkServerAddress string) error`

Reply:
This isn't resolved, and we don't have to (and probably shouldn't) take pointers for EUIs here